Sources/Web3Core/KeystoreManager/BIP44.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  BIP44.swift
+//  Created by Alberto Penas Amor on 15/12/22.
+//
+
+import Foundation
+
+public protocol BIP44 {
+    /**
+    Derive an ``HDNode`` based on the provided path. The function will throw ``BIP44Error.warning`` if it was invoked with throwOnWarning equal to`true` and the root key doesn't have a previous child with at least one transaction. If it is invoked with throwOnError equal to `false` the child node will be derived directly using the derive function of ``HDNode``. This function needs to query the blockchain history when throwOnWarning is `true`, so it can throw network errors.
```

Comment:
While this is unnecessary it's better to split lines not much more than 80-120 chars. As for now I have had to toggle on line wrapping to be able to see the whole comment. I mean the other folk developer who decide to read the code within en editor would suffer too.

Disclaimer: Since this is doesn't affects code itself it's not an restriction to approve this PR. It could be fixed within any given patch later on.

Reply:
I have added line jumps on 120

Reply:
@yaroslavyaroslav could you take a look? I lose the check